openff/benchmark/geometry_optimizations/compute.py

```diff
@@ -15,7 +15,7 @@
 #logging.disable(logging.WARNING) 
 
 from qcportal import FractalClient
-
+from openforcefield.topology import Molecule
```

Comment:
Please only place this in a method that needs it. This library is a bit optimized for the CLI calls, which can be slow if even e.g. `openff-benchmark optimize status` has to import the toolkit.